build.yaml

```diff
@@ -1120,7 +1121,35 @@ steps:
     - base_image
     - service_base_image
     - ci_utils_image
+    - batch_worker_image
     - deploy_batch
+ - kind: runImage
+   name: test_batch_copy_files
+   image:
+     valueFrom: batch_image.image
+   script: |
+     export HAIL_DEPLOY_CONFIG_FILE=/deploy-config/deploy-config.json
+     export PROJECT={{ global.project }}
+     # FIXME parallelize this
+     python3 -m pytest --log-date-format="%Y-%m-%dT%H:%M:%S" --log-format="%(asctime)s %(levelname)s %(name)s %(filename)s:%(lineno)d:%(funcName)s %(message)s" --log-cli-level=INFO -s -vv --instafail /io/test/test_copy_files.py
+   inputs:
+    - from: /repo/batch/test
+      to: /io/
+   timeout: 1200
+   resources:
+     cpu: "1"
+   secrets:
+    - name: gce-deploy-config
+      namespace:
+        valueFrom: default_ns.name
+      mountPath: /deploy-config
+   dependsOn:
+    - default_ns
+    - batch_pods_ns
+    - copy_files
+    - base_image
+    - service_base_image
+    - batch_image
```

Comment:
We just need `batch_image` and  `copy_files`, we don't reference any of the other things